cruise-control-core/src/main/java/com/linkedin/cruisecontrol/detector/metricanomaly/PercentileMetricAnomalyFinder.java

```diff
@@ -34,6 +34,8 @@
  */
 public abstract class PercentileMetricAnomalyFinder<E extends Entity> implements MetricAnomalyFinder<E> {
   private static final Logger LOG = LoggerFactory.getLogger(PercentileMetricAnomalyFinder.class);
+  // Ensure that no metric anomaly is generated unless the upper percentile metric value is a significant metric value.
+  private static final double SIGNIFICANT_METRIC_VALUE_THRESHOLD = 1;
```

Comment:
My only concern is that we hard coding threshold to be 1 here, which contains an assumption of the metrics we are monitoring. It is reasonable with the current metrics we have, but we need to be careful in the future for new metrics which extend from this abstract class.